### ISSUE TITLE
Including jsonapi object

### ISSUE THREAD

**Issue by Calamitous** [fotinakis/jsonapi-serializers#81]
I've been digging around looking for a way to add the top-level "jsonapi object" (http://jsonapi.org/format/#document-jsonapi-object) per the spec, so that we can easily see what version of json:api a service is using (especially now that the version 1.1 spec is under discussion).

Is there a way to output this object with the current system, or are there plans to add it?  I'd be happy to submit a pull request if this fits within the scope and intent of the project.

Thanks!

**Comment by fotinakis** [comment]
Interesting! There is not an official method for this yet. You could, of course, easily wrap all jsonapi-serializers access into a more top-level method and add it to the output there (see `serialize_model` and `serialize_models` in the [Rails example](https://github.com/fotinakis/jsonapi-serializers#rails-example)).

If you'd like an official way, I'd be happy to accept a PR where you implement a `jsonapi` top-level arg (see how `meta` is implemented, simple):
```
JSONAPI::Serializer.serialize(post, jsonapi: {version: '1.0'})
```

**Comment by Calamitous** [comment]
Alrighty, I'll take a crack at it.  :)  I just to make sure it was something not already supported before I started laying down code.

**Comment by Calamitous** [comment]
Okay, I've got a PR up: https://github.com/fotinakis/jsonapi-serializers/pull/82.  Please me me know if it's sufficient, or if I need to make any changes.  Thanks!

**Comment by fotinakis** [comment]
Released as gem v0.15.0. Thanks for contributing!

**Comment by Calamitous** [comment]
Awesome, thanks!
